docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+Alluxio is a data orchestration platform brings your data closer to compute across clusters,
+regions, clouds, and countries. This guide will go through how to set up the RAPIDS Accelerator
+for Apache Spark 3.0 with Alluxio and on premise cluster.
```

Comment:
we should probably have a section specifically for On Prem because we were going to add a seciton for Databricks right?

Reply:
ok